### DISCUSSION TITLE
essay/2024/03/07/people-saving-cost

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#179] [Comments]
# essay/2024/03/07/people-saving-cost

카카오에 처음 들어가서 리눅스와 웹서버를 다루는 일을 처음 해봤습니다. 2012년.

https://jeho.page/essay/2024/03/07/people-saving-cost.html

**Comment by qj0r9j0vc2** [comment]
헉…. 생각만 해도 끔찍한 일이네요..

**Reply by BenjaminKim** [reply]
하하, 개발 서버라 그렇게 끔찍한 일은 아닙니다.
실서버에서도 몇 번 사고친 적 있는데 그게 정말 끔찍한 일입니다.

**Comment by hyunsangwon** [comment]
마지막 문장에 감동 받았습니다.

'이 회사는 분명히 잘될 거야.
이런 사람들하고 일해서 너무 좋다.'

이래서 조직 분위기가 얼마나 중요한지 다시 한번 느껴봅니다.
좋은 글 감사합니다 :)

**Reply by BenjaminKim** [reply]
회사 생활 중 가장 좋았던 시절이네요. 😁
